### ISSUE TITLE
Include file comments in verified contracts

### ISSUE THREAD

**Issue by alsco77** [rkalis/truffle-plugin-verify#31]
### Feature Request

Include the comments on the file level for the flattened files. The flattened/verified file would ideally contain all file level comments.

Comment example
```
/**
 * @notice This is the notice
 * @dev Dev comments
 */
contract MyContract {
```

**Comment by rkalis** [comment]
Hey @alsco77, are you using the latest version of truffle-plugin-verify? Comments should be included already (see #23 and RyuuGan/sol-merger#19), but it's possible that this broke again.

**Comment by alsco77** [comment]
Hi @rkalis Thanks for the prompt response. Yes using `0.3.10`.

I tried using `sol-merger` previously - they preserve the **in**File comments, but not the files comments themselves. So I suppose then the issue is with sol-merger.

**Comment by rkalis** [comment]
So when you say files comments, do you mean the comments *outside* of a contract? So like this:

```solidity
pragma solidity 0.6.0;

// This is not included
contract MyContract {
    // This is included
    function myFunction() {
        // This is included
    }
}
// This is not included
```

**Comment by alsco77** [comment]
Exactly. I just had a poke around on the `sol-merger` codebase. I suspect the changes need to be made in [this function analyzeExports()](https://github.com/RyuuGan/sol-merger/blob/a830512f9465f3edef9698c787ce52c6e6cf1243/lib/exportsAnalyzer.ts#L26), to enable the comments preceeding the 'contract'/'node' to be included somewhere.

As for how to do that, i'm not sure.

**Comment by alsco77** [comment]
Appreciate the x-post @rkalis . Unfortunately I will just perform manual verification until this is fixed. Thanks for creating such an easy to use tool though! This is a vast improvement on previous UX for verifying contracts, starred.

**Comment by rkalis** [comment]
This should be fixed in `v0.3.11`. Feel free to reopen if there are any more issues :)
